public/app/plugins/datasource/prometheus/querybuilder/shared/parsingUtils.ts

```diff
@@ -200,3 +200,11 @@ function jsonToText(
 function nodeToString(expr: string, node: SyntaxNode) {
   return node.name + ': ' + getString(expr, node);
 }
+
+/**
+ * There aren't any spaces in the metric names, so let's introduce a wildcard into the regex for each space to better facilitate a fuzzy search
```

Comment:
So this is true for metric names, are we **certain** this is the case for label values, which is a field that accepts regex as valid input?